InstallNotes/MakeDefsLocalExamples/cineca_marconi.Make.defs.local

```diff
@@ -0,0 +1,18 @@
+DIM             = 3
+DEBUG           = FALSE
+OPT             = HIGH
+CXX             = icpc -std=c++14 -qopenmp -mkl=sequential -traditional -xCORE-AVX512
+FC              = ifort -qopenmp -mkl=sequential
+MPI             = TRUE
+OMP             = TRUE
+MPICXX          = mpiicpc -std=c++14 -qopenmp -mkl=sequential -xCORE-AVX512
```

Comment:
You've added the `-traditional` flag to the normal C++ compiler but not the MPI one. Is there a reason for that? I don't think it's needed as it appears from the modules you are using the 2018 version of the Intel compiler (it was only required for 2019 and later).